bumble/device.py

```diff
@@ -3541,11 +3543,9 @@ def on_advertising_set_termination(
         connection_handle,
         number_of_completed_extended_advertising_events,
     ):
+        # Legacy advertising set is also one of extended advertising sets.
```

Comment:
Yes, indeed! I think we could also remove the test on line 2212 for the same reason.

Reply:
Done